cfme/fixtures/cli.py

```diff
@@ -302,134 +300,219 @@ def ha_appliances_with_providers(ha_multiple_preupdate_appliances, app_creds):
+    configure_primary_replication_node(apps0, pwd)
+    configure_standby_replication_node(apps1, pwd, app0_ip)
 
-    # Configure secondary (standby) replication node
-    interaction = SSHExpect(apps1)
-    interaction.send('ap')
-    interaction.answer('Press any key to continue.', '', timeout=20)
-    interaction.answer('Choose the advanced setting: ',
-                       '6' if apps1.version < '5.10' else '8')
-    # 6/8 for Configure Database Replication
-    # Configure Server as Standby
-    interaction.answer('Choose the database replication operation: ', '2')
-    interaction.answer('Choose the encryption key: |1| ', '2')
-    interaction.send(app0_ip)
-    interaction.answer('Enter the appliance SSH login: |root| ', '')
-    interaction.answer('Enter the appliance SSH password: ', pwd)
-    interaction.answer('Enter the path of remote encryption key: |/var/www/miq/vmdb/certs/v2_key|',
-                       '')
-    interaction.answer('Choose the standby database disk: |1| ',
-                       '1' if apps1.version < '5.10' else '2')
-    # "Enter " ... is on line above.
-    interaction.answer('.*the number uniquely identifying this '
-                       'node in the replication cluster: ',
-                       '2')
-    interaction.answer('Enter the cluster database name: |vmdb_production| ', '')
-    interaction.answer('Enter the cluster database username: |root| ', '')
-    interaction.answer('Enter the cluster database password: ', pwd)
-    interaction.answer('Enter the cluster database password: ', pwd)
-    interaction.answer('Enter the primary database hostname or IP address: ', app0_ip)
-    interaction.answer('Enter the Standby Server hostname or IP address: |.*|', app1_ip)
-    interaction.answer(r'Configure Replication Manager \(repmgrd\) for automatic '
-                       r'failover\? \(Y/N\): ', 'y')
-    interaction.answer(r'Apply this Replication Server Configuration\? \(Y/N\): ', 'y')
-    interaction.answer('Press any key to continue.', '', timeout=5 * 60)
-
-    # Configure automatic failover on EVM appliance
-    interaction = SSHExpect(apps2)
-    interaction.send('ap')
-    interaction.answer('Press any key to continue.', '', timeout=20)
-    interaction.expect('Choose the advanced setting: ')
-
-    with waiting_for_ha_monitor_started(apps2, app1_ip, timeout=300):
-        # Configure Application Database Failover Monitor
-        interaction.send('8' if apps2.version < '5.10' else '10')
-        interaction.answer('Choose the failover monitor configuration: ', '1')
-        # Failover Monitor Service configured successfully
-        interaction.answer('Press any key to continue.', '')
+    configure_automatic_failover(apps2, primary_ip=None)
 
     # Add infra/cloud providers and create db backup
     provider_app_crud(VMwareProvider, apps2).setup()
     provider_app_crud(EC2Provider, apps2).setup()
+    return appliances
+
+
+def configure_primary_replication_node(appl, pwd):
```

Comment:
The configure methods are very similar to the reconfigure methods, but there are slight differences.